packaging/suse/release/release.sh

```diff
@@ -67,8 +67,14 @@ update_package() {
     echo "WARNING: _service file has not been changed"
   fi
 
+  echo "Remove previous tarballs"
+  old_tarball=$(ls *.tar.gz)
+  $OSC rm $old_tarball
```

Comment:
Why not simply call `$OSC rm $(ls *.tar.gz)`? The variable `old_tarball` isn't used anywhere else in the code.